src/buildblock/find_STIR_config.cxx

```diff
@@ -0,0 +1,36 @@
+
+#include "stdlib.h"
+#include "stir/find_STIR_config.h"
+#include "stir/info.h"
+#include "stir/config.h"
+#include <iostream>
+#include <fstream>
+#include <cstdlib>
+
+START_NAMESPACE_STIR
+
+//! the following returns the path of the configuration file "filename we are looking for
+std::string find_STIR_config_file(const std::string& filename){
+    
+    std::string dir;
+    dir = get_STIR_config_dir(); //STIR_CONFIG_DIR;
+    std::ifstream file(dir+"/"+filename);
+    if (!file)
+//        info("Using config file from "+dir);
+//    else
+        error("Could not open "+dir+"/"+filename);
+    
+    return dir+"/"+filename;
+
+}
+//! The following gets the STIR configuration directory 
```

Comment:
doxygen needs to be in `.h` but also reformulated. maybe
```
/*! \brief gets the STIR configuration directory 
  
 \return a string with the (full) path of the directory where STIR configuration files are found

 First checks an environment variable `STIR_CONFIG_DIR`. If that isn't set, it returns the value of the `STIR_CONFIG_DIR` CMake variable set at build time (with a default location in the installation directory).

\ingroup ancillary
```

Reply:
> please check previous comment for suggested text

you mean this comment? I thought it was generic for the class

Reply:
yes. I'm confused by your reply though. there is no class. These are functions.